{"name": "test_case", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 7", "trace": "self = <test.demo1.test_pytest-allure2.Test_case object at 0x03BA67B0>\n\n    @allure.story(\"定义场景\")\n    def test_case(self):\n        with allure.step(\"定义步骤\"):\n            C =case().add(a,b)\n        with allure.step(\"断言\"):\n>           assert C == 7\nE           assert 3 == 7\n\ntest_pytest-allure2.py:13: AssertionError"}, "steps": [{"name": "定义步骤", "status": "passed", "start": 1585047678209, "stop": 1585047678209}, {"name": "断言", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 7\n", "trace": "  File \"d:\\python_work\\test\\demo1\\test_pytest-allure2.py\", line 13, in test_case\n    assert C == 7\n"}, "start": 1585047678209, "stop": 1585047678209}], "attachments": [{"name": "stdout", "source": "10a5a692-a123-4783-afaf-8ec2bb2166d8-attachment.txt", "type": "text/plain"}], "start": 1585047678208, "stop": 1585047678209, "uuid": "7fed826e-46ca-43ab-a123-87febab678a9", "historyId": "70b3cc93fe42ead085744f5cde99956a", "testCaseId": "0688394cee7576038edfe5b52c8cfaa8", "fullName": "test_pytest-allure2.Test_case#test_case", "labels": [{"name": "feature", "value": "定义功能"}, {"name": "story", "value": "定义场景"}, {"name": "suite", "value": "test_pytest-allure2"}, {"name": "subSuite", "value": "Test_case"}, {"name": "host", "value": "law"}, {"name": "thread", "value": "7608-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest-allure2"}]}